### ISSUE TITLE
Took down MODX

### ISSUE THREAD

**Issue by flinx777** [alanpich/tvImagePlus#64]
I just tried to install this plugin and as explained in my previous Issue I just posted, I tried to remove it since it wasn't installing.  Then when I navigated in MODX, it was causing errors thus messing up my MODX install:

http://screencast.com/t/IQeHqTe5

Again, this is in MODX 2.2.10.  I had to go through and manually delete those files off my server to get MODX working again.

**Comment by alanpich** [comment]
Hmmm... thats not good. Apologies for that. Iasue seems to be caused by plugin firing halfway through the build process. 

Was this an upgrade or a fresh install of 3.0.1?

**Comment by smashbrand-kris** [comment]
Fresh install of this plugin.  It had never been used on this MODX install.  It got worse.  I had to finally dig into the db and remove all instances of the plugin.  Just doing a removal via package management and deleting the files off the server manually didn't fix the problem.  Once it dug in, it really started doing a number on this site.

**Comment by MokoJumbie** [comment]
Incidentally, are you running PHP 5.3+? I had a similar issue and discovered that my host was on PHP 5.2.x.

**Comment by flinx777** [comment]
Nope…just PHP 5.2.x.  My biggest concern is how badly it destroyed a project I was working at the time.  I had to go through the db and forcefully remove the code, otherwise it kept inserting files on the server that took down the manager.  Lost several hours that day having to clean up the server and db.  Definitely needs some type of warning regarding 5.2.x

**Comment by MokoJumbie** [comment]
Yeah, I found out later that PHP 5.3.x is a requirement as stated here: http://alanpich.github.io/tvImagePlus/#toc_0

I just configured my server to serve *.php files using PHP 5.3 instead of the server default.

FYI, there is an outstanding issue requesting that 5.3 is checked as a dependency before proceeding with the install:  #62
